builds/fabric8-che/assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/codenvy/rh-che.properties

```diff
@@ -3,3 +3,6 @@
 che.keycloak.oso.endpoint=NULL
 che.keycloak.github.endpoint=NULL
 che.keycloak.disabled=true
+che.keycloak.auth-server-url=https://sso.openshift.io/auth
```

Comment:
FYI you should use _ as string separator instead of -
like on https://github.com/eclipse/che/blob/master/assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/codenvy/che.properties#L104

This is because we allow overriding properties with env variables
and we can use same pattern than on Che